artemis-protocols/artemis-amqp-protocol/src/test/java/org/apache/activemq/artemis/protocol/amqp/broker/AMQPMessageTest.java

```diff
@@ -2231,14 +2232,14 @@ private boolean isEquals(Section left, Section right) {
          byte[] leftArray = leftValue.getValue().getArray();
          byte[] rightArray = rightValue.getValue().getArray();
 
-         if (leftArray == null && leftArray == null) {
+         if (leftArray == null && rightArray == null) {
```

Comment:
@tabish121 I figure this is what you've meant to say here (error prone screams if there are duplicated subexpressions in ifs, like this)